filetests/isa/x86/relax_branch.clif

```diff
@@ -0,0 +1,119 @@
+test binemit
+set opt_level=best
+set avoid_div_traps
+set baldrdash_prologue_words=3
+set allones_funcaddrs
+set probestack_enabled=false
+target x86_64 haswell
+
+function u0:2691(i32 [%rdi], i32 [%rsi], i64 vmctx [%r14]) -> i64 uext [%rax] baldrdash {
```

Comment:
Can you add a comment mentioning what this test is testing? Perhaps link to the issue, and point out that the key feature of the code here is an entry block that gets zero bytes of encoding.